spectral-octal-number-issue.md

```diff
@@ -0,0 +1,109 @@
+# Non-Octal Numbers In Unquoted Strings
+
+## Issue Description
+
+When linting OpenAPI 3.x.x specifications, Spectral lint is not appropriately
+parsing strings per the YAML specification. This issue presents itself when
+strings that are made up of numbers that begin with the digit `0`, and contain
+the digits `8` and `9`, are not quoted in the OpenAPI specification.
```

Comment:
"`8` or `9`"? It's even when they are quoted, right?

Reply:
If the invalid numbers are quoted then spectral treats them as strings.

Reply:
If the field values are `'08'` or `'09'` then Spectral will read these as strings.

Reply:
Oh right. That was because `json-refs` was removing the quotes, which caused Spectral to break.